doc/src/guides/solving/index.md

```diff
@@ -21,19 +21,21 @@ Learn how to use SymPy computer algebra system to:
 |  {func}`Solve a matrix problem algebraically <sympy.matrices.matrices.MatrixBase.solve>`                    | $ \left[\begin{array}{cc} 1 & 1\\1 & -1\end{array}\right] \left[\begin{array}{cc} x\\y\end{array}\right] = \left[\begin{array}{cc} 2\\0\end{array}\right] $ | $ \left[\begin{array}{cc} x\\y\end{array}\right] = \left[\begin{array}{cc} 1\\1\end{array}\right]$  |
 |  [ Solve an inequality algebraically ](../../modules/solvers/inequalities.rst)                      | $ x^2 < 4 $ | $ -2 < x < 2 $                                                                                                        |
 |  [ Solve a system of inequalities algebraically ](../../modules/solvers/inequalities.rst)           | $ x^2 < 4, x > 0 $ | $ 0 < x < 2 $                                                                                                  |
-| [ Solve (find the roots of) a polynomial algebraically ](../../modules/polys/basics.rst)                       | $ x^2 - x = 0 $ | $ x \in \{0, 1\} $                                                                                                |
+| [ Solve (find the roots of) a polynomial algebraically ](solve-polynomial.md)                       | $ ax^2 + bx + c = 0 $ | $ x = \frac{-b\pm\sqrt{b^2 - 4ac}}{2a} $                                                                                                |
 |  [ Solve a Diophantine equation (find integer solutions to a polynomial equation) algebraically ](../../modules/solvers/diophantine.rst)             | $x^2 - 4xy + 8y^2 - 3x + 7y - 5 = 0 $ | $ \{(x = 2, y = 1), (x = 5, y = 1)\}$                                                                                  |
 
-Note: SymPy has a function called {func}`~.solve`
-which is designed to find the roots of an equation or system of equations.
-SymPy {func}`~.solve` may or may not be what you need for a particular problem,
-so we recommend you use the links on this page to learn how to "solve" your problem.
-And while a common, colloquial expression is, for example, ["solve an integral"](../../modules/integrals/integrals.rst),
-in SymPy's terminology it would be ["evaluate an integral."](../../modules/integrals/integrals.rst)
+Note: SymPy has a function called {func}`~.solve` which is designed to find the
+roots of an equation or system of equations. SymPy {func}`~.solve` may or may
```

Comment:
Standard terminology is that a *function* has "roots" whereas an *equation* has "solutions".